### DISCUSSION TITLE
Celery docs - down

### DISCUSSION THREAD

**Discussion by moaddib666** [celery/celery#7402] [Q&A]
Hi, as far as I can see:
https://docs.celeryproject.org/
https://celeryproject.org/
are down. 

Is the https://docs.celeryq.dev/en/stable/copyright.html valid new site ?

Where we can track updates for now ? 

Also, I'd tray to send  a message to [ask@celeryproject.org] with no luck, in case there are any chance to contact site owner please ping me.

Regards Max. N

**Comment by auvipy** [comment]
check the repo wesite link, we got new domain
